### PR TITLE
Update README for intelligent sampling and batching

### PR DESCRIPTION
**Pull request by flands**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -307,6 +292,56 @@ The collector also serves as a control plane for agents/clients by supplying
...
+
+### <a name="tail-sampling"></a>Intelligent Sampling
+
+```yaml
+sampling:
+  mode: tail
+  # amount of time from seeing the first span in a trace until making the sampling decision
+  decision-wait: 10s
+  # maximum number of traces kept in the memory
+  num-traces: 500
```
Comment (pjanotti) [inline README.md]: nit: put a higher number 10000 seems reasonable

File: README.md [hunk 2]
```diff
@@ -307,6 +292,56 @@ The collector also serves as a control plane for agents/clients by supplying
...
+    # user-defined policy name
+    my-rate-limiting:
+      # exporters the policy applies to
+      exporters:
+        - jaeger
+        - omnition
+      policy: rate-limiting
+      configuration:
+        spans-per-second: 10
+#      policy: string-tag-filter
```
Comment (pjanotti) [inline README.md]: I think that these comments here will be confusing. If we want to show those here as example we should have other user defined policies.

File: README.md [hunk 3]
```diff
@@ -307,6 +292,56 @@ The collector also serves as a control plane for agents/clients by supplying
...
+  policies:
+    # user-defined policy name
+    my-rate-limiting:
+      # exporters the policy applies to
+      exporters:
+        - jaeger
+        - omnition
+      policy: rate-limiting
+      configuration:
+        spans-per-second: 10
```
Comment (pjanotti) [inline README.md]: nit: 100 or 1000 :)

File: README.md (hunk 1 above)
Comment (pjanotti) [inline README.md]: nit: use a higher number 10K seems reasonable here

File: README.md (hunk 2 above)
Comment (pjanotti) [inline README.md]: I think these here as a comment are confusing. Either we added user defined policies and use them or do not include them.

File: README.md (hunk 3 above)
Comment (pjanotti) [inline README.md]: nit: 100 seems a better number.

File: README.md (hunk 2 above)
Comment (sjkaris) [reply README.md]: This can go in the receivers/ readme I think

Comment (pjanotti) [reply README.md]: Good idea @sjkaris and we can add all policies currently available.

**Comment by flands** [comment]
@pjanotti @sjkaris addressed feedback -- please check again

File: README.md [hunk 4]
```diff
@@ -307,6 +292,56 @@ The collector also serves as a control plane for agents/clients by supplying
...
+        values:
+          - value1
+          - value2
+    my-numeric-tag-filter:
+      exporters:
+        - jaeger
+        - omnition
+      policy: numeric-tag-filter
+      configuration:
+        tag: <tag-name>
```
Comment (pjanotti) [inline README.md]: nit: do not use `<` and `>` on the tag name, use tag2 or something like that.

File: exporter/exporterparser/README.md [hunk 5]
```diff
@@ -2,12 +2,55 @@ A variety of exporters are available to the OpenCensus Service (both Agent and C
...
+    # user-defined policy name
+    my-rate-limiting:
+      # exporters the policy applies to
+      exporters:
+        - jaeger
+        - omnition
+      policy: rate-limiting
+      configuration:
+        spans-per-second: 1000
+#      policy: string-tag-filter
```
Comment (pjanotti) [inline exporter/exporterparser/README.md]: I favor removing the comments, I can add some examples with all other policies and their configuration to other location.

**Comment by flands** [comment]
@pjanotti addressed -- check now

**Pull request merged**